src/files-card-body.tsx

```diff
@@ -273,14 +274,38 @@ export const FilesCardBody = ({
             }
         };
 
-        const hasNoKeydownModifiers = (event: KeyboardEvent) => {
+        const hasNoModifiers = (event: KeyboardEvent) => {
             return !event.shiftKey && !event.ctrlKey && !event.altKey && !event.metaKey;
         };
 
+        const dirNavigationModifiers = (e: KeyboardEvent) => {
+            if (isApple) {
+                // use both meta and alt key on apple devices to support external non-apple keyboards
+                return !e.ctrlKey && !e.shiftKey && (e.altKey || e.metaKey);
+            } else {
+                return !e.ctrlKey && !e.shiftKey && e.altKey && !e.metaKey;
+            }
+        };
+
+        // Modifiers used for copy, paste and select all shortcuts
+        const editingModifiers = (e: KeyboardEvent) => {
+            // Keep standard text editing behavior by excluding input fields
+            if ((e.target instanceof HTMLInputElement)) {
+                return false;
+            }
+
+            if (isApple) {
+                // use both meta and ctrl key on apple devices to support external non-apple keyboards
+                return !e.shiftKey && !e.altKey && (e.ctrlKey || e.metaKey);
+            } else {
+                return !e.shiftKey && !e.altKey && e.ctrlKey && !e.metaKey;
+            }
```

Comment:
Seen this somewhere before, can't we make it re-usable in `src/common.ts`

Reply:
Not really sure as there are three different key conditions in the `return` expr. At least in this file it is extracted to a function as it is used on multiple keybinds in this file.